components/mpas-ocean/src/shared/mpas_ocn_tendency.F

```diff
@@ -658,7 +658,7 @@ subroutine ocn_tend_tracer(tendPool, statePool, forcingPool, &
       do k = 1,nVertLevels
          normalThicknessFlux(k,iEdge) = &
                  normalTransportVelocity(k,iEdge)* &
-                 layerThickEdge(k, iEdge)
+                 layerThickEdgeFlux(k, iEdge)
```

Comment:
@dengwirda Pointing you to the use of `layerThickEdgeFlux` in `ocn_tend_tracer` for your review.